src/library.js

```diff
@@ -783,16 +785,16 @@ const createPilingJs = (rootElement, initOptions = {}) => {
       itemSizeRange[1] <= 1
     ) {
       widthRange = [
-        layout.cellWidth * itemSizeRange[0],
-        layout.cellWidth * itemSizeRange[1]
+        layout.itemWidth * itemSizeRange[0],
```

Comment:
The code from the grid where you compute the `itemWidth` and `itemHeight` should be triggered prior to this. Maybe within the function or maybe as a separate function.